### PR TITLE
Feature/dx 695 cleanup summary

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: checkout/checkin.md [hunk 1]
```diff
@@ -169,6 +167,9 @@ operation is meant to be embedded in a `<script>` element in an HTML document.
 </html>
 ```
 
+The Checkin and Payment Menu components (the two `<iframe>` elements) must be
```
Comment (asbjornu) [inline checkout/checkin.md]: Perhaps we should use `alert` for this?

**Comment by marinafrydenb** [comment]
set more of the changes into alerts. I think we can have a summary at the end of each page in the next branch because we still have to restructure the pages into step-wise implementation

File: checkout/other-features.md [hunk 2]
```diff
@@ -1290,15 +1289,30 @@ although that might be possible in the future.
...
+{% include alert.html type="neutral" icon="info" body= "Features that are not
+described in the previous sections must not be used, although they are
+available in the API. Flags that can be turned to `true` must be kept
+`false` as described in this standard setup documentation." %}
+
+{% include alert.html type="neutral" icon="info" body= "Your integration must be
+resilient to change. Properties, operations, headers, etc., that aren't
+understood in any response **must be ignored**. Failing due to a something
+occurring in a response that your implementation haven't seen before is a
+malfunction that must be fixed." %}
```
Comment (asbjornu) [inline checkout/other-features.md]: I believe the language of this alert was softened at some point. If not, let's try to soften it now.

Comment (asbjornu) [inline checkout/other-features.md]:
```suggestion
{% include alert.html type="neutral" icon="info" body= "Your integration must be
resilient to change. Properties, operations, headers, etc., that aren't
understood in any response **must be ignored**. Don't expect a specific order of
elements. When in doubt, please follow the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle)." %}
```

**Pull request merged**